### PR TITLE
Jest config

### PR DESCRIPTION
**Pull request by kkoskelin**
This pull request details an initial effort to explicitly provide a consistent jest configuration (jest.config.js) per each of the three folders (web-api, shared, efcms-service).  More refinement for "include" and "exclude" patterns are necessary.

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: efcms-service/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  collectCoverage: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  // collectCoverageFrom: null,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: 'coverage',
+
+  // An array of regexp pattern strings used to skip coverage collection
+  coveragePathIgnorePatterns: ['/node_modules/'],
```
Comment (dknesek) [inline efcms-service/jest.config.js:30]: lambdas?

**Review by dknesek — COMMENTED** [review]
Do we have to explicitly include things that we want covered? I was under the impression that if you _don't create a test_ for a given module at all, that module is excluded from the test coverage report. Is this correct?

NM - Just saw your initial comment :)

File: efcms-service/jest.config.js (hunk 1 above)
Comment (dknesek) [reply efcms-service/jest.config.js:30]: Never mind. Just saw your initial PR commend. :)

**Comment by kkoskelin** [comment]
Yeah, there's definitely some more refinement to be done, and I'm learning more about how jest itself reports coverage -- as you say, if there are zero tests, will it report 0% coverage, or 100%? I've got to figure that out so we don't have gaping coverage holes that aren't reported.

**Comment by kkoskelin** [comment]
@codyseibert this is your baby now.

**Pull request closed without merging**